system-int-test/test/cmr/system_int_test/ingest/collection_umm_spec_validation_test.clj

```diff
@@ -112,7 +112,7 @@
                            (data-umm-cmn/additional-attribute {:Name "bool2" :DataType "BOOLEAN" :Value true})]})
                        {:allow-failure? true})]
         (is (= {:status 422
-                :errors ["object has missing required properties ([\"CollectionProgress\",\"DataCenters\",\"Platforms\",\"ProcessingLevel\",\"ScienceKeywords\",\"TemporalExtents\"])"]}
```

Comment:
I am confused about why you had to make these test changes - CollectionProgress should still be required, right?